ports/cpprestsdk/portfile.cmake

```
@@ -18,6 +18,11 @@ if(NOT VCPKG_TARGET_IS_UWP)
         -DWEBSOCKETPP_CONFIG_VERSION=${WEBSOCKETPP_PATH})
 endif()
 
+if ("winpplx" IN_LIST FEATURES)
+    list(APPEND OPTIONS
+        -DCPPREST_PPLX_IMPL="winpplx")
```

This is cmake code. Quote the whole expession or nothing,
```suggestion
        -DCPPREST_PPLX_IMPL=winpplx)
```